src/main/java/com/moneylendingapp/controllers/UserController.java

```diff
@@ -0,0 +1,26 @@
+package com.moneylendingapp.controllers;
+
+
+import com.moneylendingapp.annotations.CurrentUser;
+import com.moneylendingapp.dto.responses.UserModel;
+import com.moneylendingapp.entities.User;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/api/v1/user")
```

Comment:
It is better to use plurals for resources when possible, you can check [this](https://blog.dreamfactory.com/best-practices-for-naming-rest-api-endpoints/#:~:text=Using%20nouns%20for%20naming%20URIs,admin%20for%20the%20administrative%20user).